processors/firehose_to_es_processor/deploy.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+TEMPLATE_DIR=${PROJECT_ROOT}/config/iam-policy-templates
+
+aws lambda create-function \
+  --cli-input-json "$(cat ${TEMPLATE_DIR}/firehose-cwl-log-processor-lambda-deployment.json | envsubst '$ES_ENDPOINT $ACCOUNT_ID' | sed s/\"/\\\"/g)"
```

Comment:
You could move the function creation and lambda deployment template to terraform too. Then you would only need to use `aws lambda update-function-code`. This would also eliminate the need for these envsubst shenanigans I introduced.